frontend/src/styles/_uswds-theme-custom-styles.scss

```diff
@@ -383,20 +383,35 @@ button.usa-pagination__button.usa-button {
   .usa-nav__submenu {
     right: 0;
   }
+  .usa-nav__primary {
+    .mobile-nav-dropdown-uncollapsed-override {
+      button:hover {
+        background-color: white;
+      }
+    }
+  }
```

Comment:
To keep all the `.mobile-nav-dropdown-uncollapsed-override` stuff together, I think you could move this inside the rule. 

```
.mobile-nav-dropdown-uncollapsed-override {
  ... // all the default rules
  @include at-media("desktop") {
    button:hover {
      background-color: white;
    }
  }
}
```